content/_data/changelogs/weekly.yml

```diff
@@ -5620,7 +5620,60 @@
     message: |-
       Fix spotbugs reported concurrency issue using an AtomicInteger in SCMDescriptor#generation.
 
+- version: '2.210'
+  date: 2019-12-22
+  changes:
+
+  # Major BUGs and regressions
+
+  - type: major bug
+    pull: 4400
+    issue: 60513
+    message: |-
+      Resolve AtomicInteger and class-filter warnings in startup log.
+    authors:
+      - StefanSpieker
+  - type: major bug
+    pull: 4394
+    issue: 59107
+    message: |-
+      User is no longer logged out when authenticating another user.
```

Comment:
This text barely helps, because all the context about queue authentication is lost.
It suggests a severe issue while it is not